lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -155,6 +181,26 @@ sub _check_domain {
     return ( $dn, { status => 'ok', message => 'Syntax ok' } );
 }
 
+$json_schemas{validate_syntax} = joi->object->strict->props(
+    domain => $zm_validator->domain_name->required,
+    ds_info => joi->array->strict->items(
+        $zm_validator->ds_info
+    ),
+    nameservers => joi->array->strict->items(
+        $zm_validator->nameserver
+    ),
+    advanced => joi->boolean,
+    ipv4 => joi->boolean,
+    ipv6 => joi->boolean,
+    profile => $zm_validator->profile_name,
+    client_id => $zm_validator->client_id,
+    client_version => $zm_validator->client_version,
+    user_ip => $zm_validator->ip_address,
+    user_location_info => $zm_validator->location,
+    config => joi->string,
+    priority => $zm_validator->priority,
+    queue => $zm_validator->queue
```

Comment:
There's no queue parameter in the API documentation. Could you either add it there or remove it here?

Reply:
Done